src/stories/NveButton.stories.ts

```diff
@@ -1,11 +1,18 @@
-import '../nve-button';
-import '../nve-spinner';
+import '../components/nve-button/nve-button';
+import '../components/nve-spinner/nve-spinner';
 import { NveButton } from './NveButton';
 
 export default {
   title: 'NveButton', // Title for the component in Storybook
   component: 'nve-button',
-  render: (args: { theme: string; variant: string; size: string; disabled: boolean; loading: boolean; outline: boolean; }) => NveButton(args),
+  render: (args: {
+    theme: string;
```

Comment:
Har vi en property som heter theme nå?

Reply:
vi har ikke det, men vi trenger det i storybook. noen props som vises i storybook skal kun brukes for å visualisere alternativer på en komponent.